plugins/Jobs.cpp

```diff
@@ -252,6 +259,22 @@ bool BedrockPlugin_Jobs::processCommand(BedrockNode* node, SQLite& db, BedrockNo
         // If no priority set, set it
         int priority = request.isSet("priority") ? request.calc("priority") : JOBS_DEFAULT_PRIORITY;
 
+        string safeParentJobID = SQ("NULL");
+        if (!request["parentJobID"].empty()) {
+            safeParentJobID = SQ(request["parentJobID"]);
+            SQResult result;
+            if (!db.read("SELECT 1 "
+                         "FROM jobs "
+                         "WHERE jobID = " + safeParentJobID + ";",
+                         result)) {
+                throw "502 Select failed";
```

Comment:
AFAIK this only returns false when the query syntax is wrong or the table or columns don't exist.